### PR TITLE
Support for smoke and co detectors.

### PR DESCRIPTION
**Pull request by w1ll1am23**
@rbflurry I think this should do it, can you test? Download my fork, checkout the `smoke_and_co` branch and run the below script from inside of /python-wink/src/ Add your access token of course. You should get output for two sensors, CO and Smoke.

``` python
#!/usr/bin/python3

import pywink
import sys
import json

pywink.set_bearer_token('PUT YOUR ACCESS TOKEN HERE')

devices = pywink.get_smoke_and_co_detectors()
for sensor in devices:
    sys.stdout.write("Name: " + str(sensor.name()) + "\n")
    sys.stdout.write("ID: " + str(sensor.device_id()) + "\n")
    sys.stdout.write("Available: " + str(sensor.available) + "\n")
    sys.stdout.write("Battery: " + str(sensor.battery_level) + "\n")
    try:
        sys.stdout.write("Smoke: " + str(sensor.smoke_detected_boolean()) + "\n")
    except:
        pass
    try:
        sys.stdout.write("CO: " + str(sensor.co_detected_boolean()) + "\n")
    except:
        pass
sys.stdout.flush()
```

### CODE REVIEW

**Comment by rbflurry** [comment]
I might have done something wrong in the setup but this is the error I get.

`Traceback (most recent call last):
  File "./test.py", line 9, in <module>
    devices = pywink.get_sensors() # get_smoke_and_co_detectors()
  File "/root/test/python-wink/src/pywink/api.py", line 134, in get_sensors
    return get_devices(device_types.SENSOR_POD)
  File "/root/test/python-wink/src/pywink/api.py", line 199, in get_devices
    response_dict = wink_api_fetch()
  File "/root/test/python-wink/src/pywink/api.py", line 188, in wink_api_fetch
    response = requests.get(arequest_url, headers=API_HEADERS)
  File "/usr/lib/python2.7/dist-packages/requests/api.py", line 55, in get
    return request('get', url, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/api.py", line 44, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/sessions.py", line 455, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/sessions.py", line 558, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/adapters.py", line 385, in send
    raise SSLError(e)
requests.exceptions.SSLError: [Errno 1] _ssl.c:510: error:14094438:SSL routines:SSL3_READ_BYTES:tlsv1 alert internal error`

**Comment by w1ll1am23** [comment]
Looks like your running it with python2, try using python3. Your python is
probably linked to python2 so you will need to explicitly call python3.

**Comment by rbflurry** [comment]
Great! started the script with python3 and now it runs without errors but gives no output. (I changed the access token to something invalid and it failed so the script is doing something)

Just to be certain (sorry I am not too familiar with git)
I ran 
git clone https://github.com/w1ll1am23/python-wink.git
then cd /python-wink
then  git checkout smoke_and_co
then created the test script in the src folder and added my access token.

**Comment by w1ll1am23** [comment]
Well crap, sorry copy the script above again and try again. I fixed a typo.

**Comment by rbflurry** [comment]
Bingo! 

root@homeass:~/test/python-wink/src# python3 ./test.py
Name: Hallway Smoke Detector smoke_detected
ID: 73142+smoke_detected
Available: True
Battery: 0.9
Smoke: False
Name: Hallway Smoke Detector co_detected
ID: 73142+co_detected
Available: True
Battery: 0.9

**Comment by rbflurry** [comment]
Except it didn't give me a readout for CO like it did for "Smoke: False"

Perhaps test_activated might be of value as well for testing automation? without setting things on fire?

Thank you so much for looking at this!

**Comment by w1ll1am23** [comment]
Yeah, I was wondering about that test thing. When you run a test I wonder if the smoke and co will return True? Might happen too quick to test though. Also there was another typo in the script that is why you didn't get the response on the co.

**Comment by rbflurry** [comment]
Works now.

root@homeass:~/test/python-wink/src# python3 test.py
Name: Hallway Smoke Detector smoke_detected
ID: 73142+smoke_detected
Available: True
Battery: 0.9
Smoke: False
Name: Hallway Smoke Detector co_detected
ID: 73142+co_detected
Available: True
Battery: 0.9
CO: False

**Comment by rbflurry** [comment]
Let me know if you need anything else. 

I guess once this gets pulled I will need to wait for HA to pull the changes as well?

**Comment by w1ll1am23** [comment]
Yeah, the HA pull request will be simple too. I'll add it in once this gets
merged.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+## 0.8.1
```
Comment (bradsk88) [inline CHANGELOG.md:3]: This should be 0.9.0 because it's the addition of a new feature.
I'll update it after the merge.

**Review by bradsk88 — APPROVED** [review]

**Comment by bradsk88** [comment]
Published! (as 0.9.0)

**Pull request merged**